src/main/webapp/image-upload-validation.js

```diff
@@ -33,8 +33,9 @@ async function validateImageUpload() {
   // calculated as MAX_IMAGE_RESOLUTION * AVERAGE_SIZE_OF_IMAGE_OF_THIS_TYPE
   // and converted to Mb.
   const MAX_SIZES = {
-    'png': Math.round(MAX_IMAGE_HEIGHT * MAX_IMAGE_WIDTH * 4 / 1024 / 1024),
-    'jpeg': Math.round(MAX_IMAGE_HEIGHT * MAX_IMAGE_WIDTH * 8.25 /
+    'image/png': Math.round(MAX_IMAGE_HEIGHT * MAX_IMAGE_WIDTH * 4 /
+        1024 / 1024),
```

Comment:
https://github.com/googleinterns/step260-2020/runs/1111448246 
Run fails due to > 80 line rule.

Reply:
Yes, thanks, it is fixed now :)